### PR TITLE
Clarify software requirements of Terraform Enterprise

### PR DESCRIPTION
**Pull request by aaron-lane**
This branch updates Software Requirements (Standalone Deployment) for Terraform Enterprise for accuracy and clarity.

## Labels

- [ ] inaccuracy
- [x] clarification
- [ ] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout
- [ ] api - requires an update to the [changelog](https://www.terraform.io/docs/cloud/api/changelog.html)

### CODE REVIEW

File: content/source/docs/enterprise/before-installing/index.html.md [hunk 1]
```diff
@@ -135,15 +135,19 @@ Some operating systems have specific configuration requirements:
...
-For other Linux distributions, check Docker compatibility:
-
-* The instance should run a supported version of Docker engine (1.7.1 or later, minimum 17.06.2-ce, maximum 19.03.11). This also requires a 64-bit distribution with a minimum Linux Kernel version of 3.10.
-    * Replicated 2.32.0 and above required when running Docker 18+.
-    * In Online mode, the installer will install Docker automatically.
-    * In Airgapped mode, Docker should be installed before you begin.
-* For _RedHat Enterprise_ and _Oracle Linux_, you **must** pre-install Docker as these distributions are [not officially supported by Docker Community Edition](https://docs.docker.com/engine/install/#server).
-
-~> **Important:** We do not recommend running Docker under a 2.x kernel.
+For other Linux distributions, check Docker requirements:
```
Comment (sudomateo) [inline content/source/docs/enterprise/before-installing/index.html.md:138]:
```suggestion
Other Linux distributions require:
```

File: content/source/docs/enterprise/before-installing/index.html.md [hunk 2]
```diff
@@ -135,15 +135,19 @@ Some operating systems have specific configuration requirements:
...
+
+- A 64-bit architecture
```
Comment (sudomateo) [inline content/source/docs/enterprise/before-installing/index.html.md:140]: This being in the "Other" section makes it read like it's not applicable to CentOS/RHEL. Perhaps we can reorder?

Comment (aaron-lane) [reply content/source/docs/enterprise/before-installing/index.html.md]: Fixed by https://github.com/hashicorp/terraform-website/pull/1980/commits/e03fb3294eee916afe42220e21ba38586b0036db.

File: content/source/docs/enterprise/before-installing/index.html.md [hunk 3]
```diff
@@ -130,21 +130,27 @@ See [Network Requirements](./network-requirements.html) for details.
...
+- A 64-bit architecture Linux-based operating system
+- Linux Kernel 3.10 or greater
+- Docker Engine 17.06.2-ce to 20.10:
+  - Docker Engine 18.01.0-ce or greater requires Replicated 2.32.0
+    or greater
+  - In Online mode, the installer will install Docker Engine
+    automatically
+  - In Airgapped mode, Docker Engine must be installed before running
+    the installer
+- The faccessat2 syscall:
```
Comment (gitrgoliveira) [inline content/source/docs/enterprise/before-installing/index.html.md:145]: is it worth to direct customers the reason for this requirement like in the release notes?
https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.14.0#faccessat2

I had no idea what the faccessat2 syscall is/does.

Comment (aaron-lane) [reply content/source/docs/enterprise/before-installing/index.html.md]: How does https://github.com/hashicorp/terraform-website/pull/1980/commits/4816fba670c6b420c4185f866d7dd08fff6417ba look?

File: content/source/docs/enterprise/before-installing/index.html.md [hunk 4]
```diff
@@ -130,21 +130,28 @@ See [Network Requirements](./network-requirements.html) for details.
 
 ### Software Requirements (Standalone Deployment)
 
-Some operating systems have specific configuration requirements:
+The following list provides the general requirements of Terraform
```
Comment (laurapacilio) [inline content/source/docs/enterprise/before-installing/index.html.md:133]:
```suggestion
Every Terraform Enterprise installation requires the following:
```

File: content/source/docs/enterprise/before-installing/index.html.md [hunk 5]
```diff
@@ -130,21 +130,28 @@ See [Network Requirements](./network-requirements.html) for details.
...
+- Docker Engine 17.06.2-ce to 20.10:
+  - Docker Engine 18.01.0-ce or greater requires Replicated 2.32.0
+    or greater
+  - Support for [Alpine 3.14](https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.14.0#faccessat2)
+    Docker images:
+    - Docker Engine 17.06.2-ce to 20.09 requires runc v1.0.0-rc93
+    - Docker Engine 20.10 or greater requires libseccomp 2.4.4
+  - In Online mode, the installer will install Docker Engine
+    automatically
+  - In Airgapped mode, Docker Engine must be installed before running
```
Comment (laurapacilio) [inline content/source/docs/enterprise/before-installing/index.html.md:147]:
```suggestion
  - In Airgapped mode, you must install Docker Engine before running
```

File: content/source/docs/enterprise/before-installing/index.html.md [hunk 6]
```diff
@@ -130,21 +130,28 @@ See [Network Requirements](./network-requirements.html) for details.
...
+    the installer
+
+The use of some operating systems comes with additional requirements:
```
Comment (laurapacilio) [inline content/source/docs/enterprise/before-installing/index.html.md:150]:
```suggestion
Some operating systems also have additional requirements:
```

**Review by laurapacilio — APPROVED** [review]
A couple nit picks for you, but thank you - I think this is a great improvement :-)

**Pull request merged**